Dockerfile

```diff
@@ -0,0 +1,43 @@
+# Start with fresh Unbuntu
+FROM ubuntu:14.04
+
+# Install dependencies
+RUN apt-get update && \
+DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+supervisor \
+git \
+apache2 \
+mysql-server \
+php5-cli php5-curl php5-mysql php5-gd php5-intl \
+libapache2-mod-php5 \
+imagemagick \
+curl
+
+EXPOSE 80 3306
```

Comment:
this is only the exposed container port, so 80 should be fine since nothing else will be running on the container.  You can set the bound local port with the docker run command options -P or -p <container_port>:<local_port>